src/client/core/utils/dom.js

```diff
@@ -225,6 +225,24 @@ function getTabIndexAttributeIntValue (el) {
     return tabIndex;
 }
 
+export function getParentsUntil (el, boundElement) {
```

Comment:
let's move it to the studio scripts, since it's not used in this repo we can kill it as obsolete during possible future refactoring